contracts/Removal.sol

```diff
@@ -316,6 +335,85 @@ contract Removal is
     });
   }
 
+  /**
+   * @notice Transfers the provided `amounts` of the specified removal `ids` directly to the Certificate contract to
```

Comment:
Can we expand on this to mention that this to cover:
* this is a one-off migration
* what a legacy certificate is
* that `amount` is denominated in NRTs.